### DISCUSSION TITLE
Unable to set cutsom title and link with model viewer

### DISCUSSION THREAD

**Discussion by devhims** [google/model-viewer#2073] [Q&A]
I'm able to pass the title and link in the explicit intent, but can't do so with model viewer. Is this intentional? It seems a pretty useful feature to me.

**Comment by elalish** [comment]
Can you link to a repro of the problem? This should be possible.

**Comment by Danfosscooling** [comment]
you can add the title and link using the src url that you provide for the respective models.

**Comment by mohanyadav** [comment]
Extending on @Danfosscooling comment

You can append the src of model-viewer with &title= and mention your title to be displayed on Sceneviewer
Example: If your src is https://cdn.glitch.com/36cb8393-65c6-408d-a538-055ada20431b/Astronaut.glb?1542147958948 then your src with title should look like this https://cdn.glitch.com/36cb8393-65c6-408d-a538-055ada20431b/Astronaut.glb?1542147958948&title=Hello

Live Demo - https://sky-marmalade-popcorn.glitch.me/

If you open the above example you can see that you will be able to type a custom title and it will be shown in AR Mode

Demo Code - https://glitch.com/edit/#!/sky-marmalade-popcorn

Video

https://user-images.githubusercontent.com/11350400/109381183-bc3be400-78fe-11eb-8c74-97d279174472.mp4

**Comment by devhims** [comment]
Thanks a bunch guys for following up on this issue! 

If I pass the `src` field as a CDN link, everything works fine. However, if I directly import an asset into my `.js` file, and then pass it as a reference, the 3d viewer stops working, but it all works fine in AR view (Scene Viewer). What do you suggest can be done here? 

Here is a snippet from my React project. 

```js

import modelGLB from '../assets/model.glb';

<model-viewer
  src={`${modelGLB}&title=${title}&link=${link}`}
>
```
